### DISCUSSION TITLE
Was there a toggle in the update section for beta releases in older versions?

### DISCUSSION THREAD

**Discussion by rp1231** [xanderfrangos/twinkle-tray#508] [General]
For some reason I remember there being an option to get beta updates as well........
Or maybe I'm confusing this feature from some other software.

**Comment by rp1231** [comment]
BTW  the new brightness OSD in the new beta looks great!

**Comment by xanderfrangos** [comment]
I don't use the beta update channel very often, but it's in the Debug menu.

**Comment by rp1231** [comment]
Aah yes found it.
I don't know how I missed it........
I went through all the settings.
